index.html

```diff
@@ -43,7 +43,7 @@
 
     <div class="top-div">
       <div class="container pb-3">
-        <ul class="list-unstyled">
+        <ul class="list-unstyled two-column fa-pull-left">
```

Comment:
As you can see we're using Bootstrap library and it would be much better to use it here. In this case you will implement two colums without additional styles and you don't need to touch `styles.css` file.

Reply:
Implemented using Bootstrap. Please review.